library/spdm_responder_lib/libspdm_rsp_key_exchange.c

```diff
@@ -264,8 +264,22 @@ return_status spdm_get_response_key_exchange(IN void *context,
 
     ptr += dhe_key_size;
 
-    result = spdm_generate_measurement_summary_hash(
-        spdm_context, FALSE, spdm_request->header.param1, ptr);
+    if (spdm_is_capabilities_flag_supported(
```

Comment:
This capability check was moved out from within the spdm_generate_measurement_summary_hash in order to eliminate the need to pass the spdm_context_t.